cli/run_test.go

```diff
@@ -234,4 +235,34 @@ func TestCommandEnvs(t *testing.T) {
 		}
 		require.Equal(t, true, env3Found)
 	}
+
+	t.Log("unset OS envs test")
+	{
+		// given
+		key := "TEST_ENV"
+		val := "test"
+		if err := os.Setenv(key, val); err != nil {
+			require.Equal(t, nil, err, "test setup: error seting env (%s=%s)", key, val)
+		}
+		env := models.EnvironmentItemModel{
+			key: val,
+			models.OptionsKey: models.EnvironmentItemOptionsModel{
+				Unset: pointers.NewBoolPtr(true),
+			},
+		}
+		require.Equal(t, nil, env.FillMissingDefaults())
+		testEnvs := []models.EnvironmentItemModel{
+			env,
+		}
+
+		// when
+		envs, err := commandEnvs(testEnvs)
+		envFmt := "%s=%s" // note: if this format mismatches elements of `envs`, test can be a false positive!
```

Comment:
`%s=%s` format for checking a system env is not the proper way to do it, especially if we do logic with that env, because you skip the possibility to check the persistence of an empty env key. For example: `TEST_ENV=`

Reply:
That's correct, but in this case I can be sure that it will match the format, because I prepared it earlier in the test.